doc/en/components/data-grid.md

```diff
@@ -4,7 +4,7 @@ _description: Infragistics' $Platform$ grid component helps you create a fast, r
 _keywords: $Platform$ Table, Data Grid, overview, $ProductName$, data binding, Infragistics
 mentionedTypes: ['Grid']
 ---
-# $Platform$ Data Grid Overview
+# $Platform$ Data Grid component| Real-Time $Platform$ tables| Infragistics 
```

Comment:
this way too long title for a topic, use one of those:
- $Platform$ Data Grid Component
- Real-Time $Platform$ Table

Reply:
@aatanasov2 

https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EcTa4fzivSBOsUHEToQGHXEBWudX1jUkc3Zbb-VmxSwCjg?rtime=R_kJiiM72Ug

Reply:
see latest commit 5

Reply:
@mddifilippo89 my recomendations for the titles are to replace the existing title tag not the topic title itself

![image](https://user-images.githubusercontent.com/72744305/123950547-7aa26f00-d9ac-11eb-8d5d-c36af9f2560c.png)

Reply:
Exisitng topic title ( H1) can remain as it is
![image](https://user-images.githubusercontent.com/72744305/123950845-cf45ea00-d9ac-11eb-8235-04e1045c1029.png)

Reply:
see commit 6